### PR TITLE
Story/display recent historic model/424

### PR DESCRIPTION
**Pull request by andrea-williams**

### CODE REVIEW

File: src/api/forecastAPI.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export interface ForecastResponse {
 }
 
 export async function getNoonForecasts(stationCodes: number[]): Promise<Forecast[]> {
-  const url = '/noon_forecasts/'
```
Comment (Kyubinhan) [inline src/api/forecastAPI.ts:32]: so I addressed this issue in my pr by changing `REACT_APP_API_BASE_URL` to `http://localhost:8080/api` in `.env`. Can you change them back?

Comment (andrea-williams) [reply src/api/forecastAPI.ts:32]: will do

Comment (Kyubinhan) [reply src/api/forecastAPI.ts:32]: My apologies for not being careful!

Comment (andrea-williams) [reply src/api/forecastAPI.ts]: no worries! it's all good :)

**Review by Sybrand — APPROVED** [review]

File: src/features/fireWeather/components/graphs/WxDataGraphToggles.tsx [hunk 2]
```diff
@@ -47,7 +50,10 @@ const WxDataToggles = ({
   setShowForecasts,
   noPastForecasts,
   showPastForecasts,
-  setShowPastForecasts
+  setShowPastForecasts,
+  noRecentHistoricModels,
+  showRecentHistoricModels,
+  setShowRecentHistoricModels
```
Comment (Kyubinhan) [inline src/features/fireWeather/components/graphs/WxDataGraphToggles.tsx:56]: I don't think we need to create these and pass down here, we can just use showModelSummaries. I will make this change

File: src/api/modelAPI.ts [hunk 3]
```diff
@@ -82,3 +92,19 @@ export async function getModelSummaries(
...
+  const url = `/models/${model}/predictions/historic/most_recent/`
+  try {
+    const { data } = await axios.post<HistoricModelsResponse>(url, {
+      stations: stationCodes
+    })
+
+    return data.predictions
+  } catch (err) {
+    throw err.toString()
+  }
```
Comment (Kyubinhan) [inline src/api/modelAPI.ts:109]: They can go away. Change is on the way

**Review by Kyubinhan — APPROVED** [review]

**Pull request merged**